external/anomaly/anomaly_classification/configs/padim/template.yaml

```diff
@@ -18,6 +18,12 @@ entrypoints:
 # Hyper Parameters
 hyper_parameters:
   base_path: ./configuration.yaml
+  parameter_overrides:
```

Comment:
what about 'stfpm'?

Reply:
I don't see any issue with 'stfpm' model which is related to POT. But I added the default parameters to the overrides section.